src/App.js

```diff
@@ -59,7 +60,15 @@ class App extends Component {
       })
     }
 
+    deleteCard = (deletedCard) => {
+      //set the state of the cards so the array of display cards no longer contains that country
+      this.setState({
+        displayCards: this.state.displayCards.filter(item => item !== deletedCard.props.country)
+        })
+    }
+
   render() {
+    if (this.state.displayCards.length < 1) {
     return (
```

Comment:
Maybe we could check if the card currently does not exist in our current state of `this.state.displayCards`. If the country is in the array, then we would render JSX that would that would inform the user that the country card already exist on the page. What are your thoughts on this?